jest-puppeteer.config.js

```diff
@@ -1,9 +1,9 @@
 module.exports = {
-    launch: {
-        dumpio: true,
-        headless: "new",
-        product: 'chrome',
-        args: ["--no-sandbox"] // to fix
-    },
-    browserContext: 'incognito'
-}
\ No newline at end of file
+  launch: {
+    dumpio: true,
+    headless: "false",
```

Comment:
```suggestion
    headless: "new",
```

restore this value to headless new

Reply:
done here 397425e1a47f87953fb0b7d2c0f4efc11e217034